pom.xml

```diff
@@ -1,135 +1,145 @@
-			<organization>Technology Brewery</organization>
-			<organizationUrl>https://technologybrewery.org</organizationUrl>
-		</developer>
-		<developer>
-			<name>Eric Konieczny</name>
-			<email>ekoniec1@gmail.com</email>
-			<organization>Technology Brewery</organization>
-			<organizationUrl>https://technologybrewery.org</organizationUrl>
-		</developer>
-	</developers>
+    <developers>
+        <developer>
+            <name>Ryan Ashcraft</name>
+            <email>dryanashcraft@gmail.com</email>
+            <organization>Technology Brewery</organization>
+            <organizationUrl>https://technologybrewery.org</organizationUrl>
+        </developer>
+        <developer>
+            <name>Eric Konieczny</name>
+            <email>ekoniec1@gmail.com</email>
+            <organization>Technology Brewery</organization>
+            <organizationUrl>https://technologybrewery.org</organizationUrl>
+        </developer>
+    </developers>
 
-	<scm>
-		<connection>scm:git:ssh://git@github.com/TechnologyBrewery/habushu.git</connection>
-		<developerConnection>scm:git:ssh://git@github.com/TechnologyBrewery/habushu.git</developerConnection>
-		<url>https://github.com/TechnologyBrewery/habushu/</url>
-		<tag>HEAD</tag>
-	</scm>
+    <scm>
+        <connection>scm:git:ssh://git@github.com/TechnologyBrewery/habushu.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/TechnologyBrewery/habushu.git</developerConnection>
+        <url>https://github.com/TechnologyBrewery/habushu/</url>
+        <tag>HEAD</tag>
+    </scm>
 
-	<properties>
-		<!-- override parent POM until it is next upgrade (arguably, this property should not be in parent at all: -->
-		<version.python.default>3.11.4</version.python.default>
-		<version.cucumber.reporting.plugin>5.8.0</version.cucumber.reporting.plugin>
-	</properties>
+    <properties>
+        <!-- override parent POM until it is next upgrade (arguably, this property should not be in parent at all: -->
+        <version.python.default>3.11.4</version.python.default>
+        <version.cucumber.reporting.plugin>5.8.0</version.cucumber.reporting.plugin>
+    </properties>
 
-	<dependencyManagement>
-		<dependencies>
-			<!-- Explicitly upgrading transitive dependency for habushu-maven-plugin
-				to mitigate vulnerability - httpcomponents:httpcore:4.4.16 -->
-			<dependency>
-				<groupId>commons-codec</groupId>
-				<artifactId>commons-codec</artifactId>
-				<version>1.16.1</version>
-			</dependency>
-		</dependencies>
-	</dependencyManagement>
+    <dependencyManagement>
+        <dependencies>
+            <!-- Explicitly upgrading transitive dependency for habushu-maven-plugin
+                to mitigate vulnerability - httpcomponents:httpcore:4.4.16 -->
+            <dependency>
+                <groupId>commons-codec</groupId>
+                <artifactId>commons-codec</artifactId>
+                <version>1.16.1</version>
+            </dependency>
```

Comment:
I: Newly upgraded cucumber-reporting-plugin solved one dependabot alert, then added a new ones.  This should take care of the new ones.